sql-jdbc/README.md

```diff
@@ -6,6 +6,15 @@ This is the driver for JDBC connectivity to a cluster running with OpenSearch SQ
 
 The driver is compatible with JDBC 4.2 specification and requires a minimum of Java 8.
 
+## BI Tool Connectors
+
+* [Power BI Desktop](../bi-connectors/PowerBIConnector/README.md)
+* [Tableau Desktop](../bi-connectors/TableauConnector/README.md)
+
+## Download and Installation
+
+The driver is available for download from an [automated CI workflow](https://github.com/opensearch-project/sql/actions/workflows/sql-jdbc-test-and-build-workflow.yml) and from [Artifacts page](https://opensearch.org/artifacts) on OpenSearch.org at the very bottom.
```

Comment:
Can you swap the ordering?  We should recommend that the JDBC driver be downloaded from the artifacts directory first.

Reply:
Also, please remove 'an' from: `The driver is available for download from an`